src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -300,19 +306,22 @@
   "ImportCustomFormat": "Import Custom Format",
   "Imported": "Imported",
   "ImportedTo": "Imported To",
+  "ImportErrors": "Import Errors",
   "ImportExistingMovies": "Import Existing Movies",
   "ImportExtraFiles": "Import Extra Files",
   "ImportExtraFilesHelpText": "Import matching extra files (subtitles, nfo, etc) after importing an movie file",
   "ImportFailed": "Import failed: {0}",
   "ImportFailedInterp": "Import failed: {0}",
   "ImportFirstTip": "Make sure that your files include the quality in their filenames. eg.",
   "ImportHeader": "Import movies you already have",
+  "ImportIncludeQuality": "Make sure that your files include the quality in their filenames. eg. {0}",
   "Importing": "Importing",
   "ImportListStatusCheckAllClientMessage": "All lists are unavailable due to failures",
   "ImportListStatusCheckSingleClientMessage": "Lists unavailable due to failures: {0}",
   "ImportListSyncIntervalHelpText": "How often Radarr syncs with your lists.",
   "ImportMechanismHealthCheckMessage": "Enable Completed Download Handling",
   "ImportMovies": "Import Movies",
+  "ImportRootPath": "Point Radarr to the folder containing all of your movies, not a specific one. eg. {0} and not {1}",
   "ImportSecondTip": "Point Radarr to the folder containing all of your movies, not a specific one. eg.",
```

Comment:
Purge this one, since we didn't ever end up using it and you replaced it above, there is also one called `andnot` purge it as well